packaging/linux/openra.6

```diff
@@ -0,0 +1,68 @@
+.TH OPENRA 6
+.SH NAME
+openra \- An open-source implementation of the Command & Conquer: Red Alert engine using .NET/Mono and OpenGL.
```

Comment:
I suggest "classic Westwood real-time strategy games like C&C Red Alert, Tiberian Dawn and Dune 2000" to not exclude Dune 2000, explain what RA stands for and avoid the Command & Conquer trademark.